src/ostorlab/serve_app/types.py

```diff
@@ -735,10 +718,10 @@ class OxoLinkInputType(graphene.InputObjectType):
 
 
 class OxoAssetInputType(graphene.InputObjectType):
-    android_file = OxoAndroidFileAssetInputType()
-    ios_file = OxoIOSFileAssetInputType()
-    android_store = OxoAndroidStoreAssetInputType()
-    ios_store = OxoIOSStoreAssetInputType()
+    android_file = graphene.List(OxoAndroidFileAssetInputType)
+    ios_file = graphene.List(OxoIOSFileAssetInputType)
+    android_store = graphene.List(OxoAndroidStoreAssetInputType)
+    ios_store = graphene.List(OxoIOSStoreAssetInputType)
```

Comment:
the name should indicate it's a list: android_files, ios_file...

Reply:
im adapting to yaml file format : 
![image](https://github.com/Ostorlab/oxo/assets/144013278/8eb2cd55-acd7-403c-81f3-32af9444adbe)